running/routing/ecan-soar-prod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: ecan-soar-prod
```

Comment:
is this still running?

Reply:
I made ingress for anything that still had pods running, so technically yes. I don’t know if it’s being used to Pugh.